pytorch_lightning/trainer/trainer.py

```diff
@@ -2474,6 +2474,40 @@ def _active_loop(self) -> Optional[Union[FitLoop, EvaluationLoop, PredictionLoop
     Logging properties
     """
 
+    @property
+    def logger(self) -> Optional[LightningLoggerBase]:
+        return self._logger
+
+    @logger.setter
+    def logger(self, new_logger: Union[LightningLoggerBase, None]) -> None:
+        if new_logger:
+            if isinstance(new_logger, LoggerCollection):
+                self._logger = new_logger
+                self._loggers = [_logger for _logger in new_logger]
```

Comment:
we can just iterate over a LoggerCollection like this?

Reply:
yes, it has `__getitem__` method defined.

Reply:
I believe so. LoggerCollection implements __getitem_() which should make it behave like an iterable. The new unit test checks for this behavior.